src/sage/combinat/posets/posets.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+# sage.doctest: needs sage.graphs sage.modules sage.plot sage.rings.finite_rings
```

Comment:
I'd suggest to use the file-level tag `sage.plot` only for files that are focused on plotting. Using block tags for the doctests of plotting is better.
Also sage.rings.finite_rings is actually not needed any more for the doctests in this file, see changes to this file in #35095